lib/client.js

```diff
@@ -311,7 +316,7 @@ const warmupLimit = 32;
  * @extends EventEmitter
  * @param {ClientOptions} options The options for this instance.
  * @example <caption>Creating a new client instance</caption>
- * const client = new Client({ contactPoints: ['192.168.1.100'] });
+ * const client = new Client({ contactPoints: ['192.168.1.100'], localDataCenter: 'datacenter1' });
```

Comment:
I chose to specify `datacenter1` in both the examples and docs because this is what Cassandra uses with default settings, and also what ccm uses for single data center.